src/base/Cellar.sol

```diff
@@ -1069,6 +1051,104 @@ contract Cellar is ERC4626, Ownable, Multicall {
 
     // ========================================= FEES LOGIC =========================================
 
+    /**
+     * @notice Stores the share price to be used as a High Watermark to calculate performance fees
+     */
+    uint256 public sharePriceHighWatermark;
+
+    /**
+     * @notice Number 0 -> 1e18 defining how much of the performance fee goes to the strategist
+     */
+    uint256 public strategistPerformanceCut = 0.75e18;
+
+    /**
+     * @notice Number 0 -> 1e18 defining how much of the platform fee goes to the strategist
+     */
+    uint256 public strategistPlatformCut = 0.75e18;
+
+    /**
+     * @notice Address to send the strategists fee shares
+     */
+    address public strategistPayoutAddress;
+
+    ///@dev resets high watermark to current share price
```

Comment:
Delete `resetHighWatermark` for now.

Also when we add it back, use `convertToAssets` to get the current share price instead of how its implemented here.